packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -88,6 +89,12 @@ class GoogleMap extends StatefulWidget {
   /// True if the map view should respond to scroll gestures.
   final bool scrollGesturesEnabled;
 
+  /// True if the map view should show zoom controls. This includes two buttons
+  /// to zoom in and zoom out. The default value is to show zoom controls.
+  ///
+  /// This field is silently ignored on iOS.
```

Comment:
Let's change this to: This is only supported on Android. And this field is silently ignored on iOS.